src/components/CompareResults/CommonGraph.tsx

```diff
@@ -0,0 +1,210 @@
+import { Chart as ChartJS, LineElement, LinearScale, TooltipItem } from 'chart.js';
+import 'chart.js/auto';
+import { Line } from 'react-chartjs-2';
+import { style } from 'typestyle';
+
+import { Spacing } from '../../styles';
+
+ChartJS.register(LinearScale, LineElement);
+
+interface GraphRun {
+  from: string;
+  oxValue: number;
+  value: number;
+}
+interface GraphData {
+  x: number;
+  y: number | string;
+}
+
+const configGraph = () => {
+  const options = {
+    plugins: {
+      datalabels: {
+        formatter: function (value: GraphData) { return value.y || null;  },
+      },
+      legend: {
+        align: 'start' as const,
+        position: 'top' as const,
+      },
+      tooltip: {
+        callbacks: {
+          label: (context: TooltipItem<'line'>) => {
+            return `${(context.raw as GraphData).y}`;
+          },
```

Comment:
I believe you'll change the tooltip later though, right? We need a better tooltip so that the user understands the data we're looking at.
(I admit I don't understand what I'm looking at currently, what is the y, what is the x???)

Reply:
You can use the `title` parameter in scales to add some legend to the scales.

For example:
```
          x: {
            ...
            title: {
              display: true,
              text: "something →",
              align: "end",
            },
          },
          y: {
            ...
            title: {
              display: true,
              text: "something →",
            },
          },
```